tests/manifest/main.rs

```diff
@@ -79,6 +84,17 @@ fn it_creates_a_package_json_provided_path() {
     assert!(utils::read_package_json(&path).is_ok());
     let pkg = utils::read_package_json(&path).unwrap();
     assert_eq!(pkg.name, "js-hello-world");
+    assert_eq!(pkg.main, "js_hello_world.js");
```

Comment:
hrm, this seems to assert that the main is not the bg file.... that's not correct right? i might pull this down and just figure out what's happening here.

Reply:
sorry for the confusion around this. i'm honestly baffled about how this bug happened but feels like something i might just need to sort out. thank you so much for all your patience and hard work. i'll pull this down and take it from here so you don't have to worry about it <3<3